samples/military_feature_schemas/Fields_Cyberspace.csv

```diff
@@ -3,6 +3,7 @@ identity,Integer,4,Standard Identity,NULLABLE,Identity,,False,,
 context,SmallInteger,2,Context,NULLABLE,Context,,False,,
 symbolset,SmallInteger,2,Symbol Set,NON_NULLABLE,,60,False,,
 symbolentity,Integer,4,Cyberspace Entity,NON_NULLABLE,Cyberspace_Entities,,False,,
+status,SmallInteger,2,Status,NULLABLE,Statuses,,False,,
```

Comment:
A feature class shouldn't have both a "status" and an "operationalcondition" - not sure which is the correct setting here or if this is a global problem